modules/ui/combobox.js

```diff
@@ -7,8 +7,7 @@ import { utilGetSetValue, utilRebind, utilTriggerEvent } from '../util';
 // It is keyed on the `value` of the entry. Data should be an array of objects like:
 //   [{
 //       value:   'string value',  // required
-//       display: 'label function' // optional, if present will be called with d3 selection
-//                                              to modify/append, see localizer's t.append
+//       display: 'label html'     // optional
```

Comment:
thanks for the pull request, but this was actually changed in 8f79932ab1259267560541bbef2ae290d68d42e6 to prevent possible html injection into UI fields. see also https://github.com/openstreetmap/iD/wiki/How-to-use-iD%27s-localizer-module#use-tappendstringid-for-adding-text-with-locale-info

Reply:
PS: I'm working on a "proper" fix for this bug right now.